Src/Support/Google.Apis.Auth.Tests/OAuth2/ExternalAccountCredentialTestsBase.cs

```diff
@@ -22,18 +22,19 @@
 using System.Net;
 using System.Net.Http;
 using System.Text;
+using System.Threading;
 using System.Threading.Tasks;
 using Xunit;
 
 namespace Google.Apis.Auth.Tests.OAuth2
 {
-    public abstract class ExternalAccountCredentialTestsBase
+    public class ExternalAccountCredentialTestsBase
```

Comment:
Hmm... does this mean that these tests end up being run multiple times, one for the base class and one for each derived class? Maybe these tests should be in a separate class instead?

Reply:
Oh yes you are right. Moving to a new class.